cmd/krew/cmd/root.go

```diff
@@ -98,6 +99,14 @@ func init() {
 	}
 
 	paths = environment.MustGetKrewPaths()
+
+	// Cobra doesn't have a way to specify a two word command (ie. "kubectl krew"), so set a custom usage template
+	// with kubectl in it. Cobra will use this template for the root and all child commands.
+	// See issue #521 for more details on the reason for this: https://github.com/kubernetes-sigs/krew/issues/521
+	customUsageTemplate := rootCmd.UsageTemplate()
+	customUsageTemplate = strings.ReplaceAll(customUsageTemplate, "{{.UseLine}}", "kubectl {{.UseLine}}")
+	customUsageTemplate = strings.ReplaceAll(customUsageTemplate, "{{.CommandPath}}", "kubectl {{.CommandPath}}")
```

Comment:
This looks like a use-case for `strings.NewReplacer(args ...string)`.

Reply:
Hmm. Good call.  I'll take a look at that